Resources/doc/examples/RssHandler.php

```diff
@@ -65,6 +65,10 @@ public function createResponse(ViewHandler $handler, View $view, Request $reques
             $content = $this->createFeed($view->getData());
             $code = Codes::HTTP_OK;
         } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```

Comment:
It depends on what we decide in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1471 but this may have to be removed